### DISCUSSION TITLE
Add Active Pen and Multitouch Support

### DISCUSSION THREAD

**Discussion by bysayem** [rustdesk/rustdesk#6537] [Feature Request]
Rust Desk is undoubtedly the only feature-rich open source remote desktop. It has cross-platform capability and supports Android and iOS clients.

However, when I use my tablet with active stylus support to control my desktop, I can use the full potential of my touch display and pen. It would be wonderful if Rust Desk could implement touch and pen support with hovering cursor. That would be the cherry on the cake.

**Comment by rustdesk** [comment]
You can sponsor via ko-fi, https://ko-fi.com/rustdesk, and then mention here.

There is 30 usd now.

https://github.com/rustdesk/rustdesk/discussions/9936#discussioncomment-13372511

**Reply by antoniosolismz** [reply]
I just chipped in with 20, hope it helps. Thank you for your work
https://Ko-fi.com/home/coffeeshop?txid=fdd78b8c-b5c5-43b6-86d9-a548604be032&mode=public&img=ogiboughtsomeone

**Comment by rp1231** [comment]
How much does it need to reach to have it implemented?
